### PR TITLE
Use JCK command template in JCK playlists 

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
- Create a macro for the common portion of JCK command line and use it in the playlists

Resolves https://github.com/AdoptOpenJDK/openjdk-tests/issues/548

Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

FYI @sophia-guo @ShelleyLambert

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Testing `sanity.jck` with jck11 on Linux: Test_grinder/job/Grinder/6235/console

Testing `sanity.jck` with jck14 on Windows: Test_grinder/job/Grinder/6239/console

Testing `extended.jck` with jck8 on s390 Linux: Test_grinder/job/Grinder/6238/console

File: jck/jck.mk [hunk 1]
```diff
@@ -58,3 +58,12 @@ ifndef JCK_ROOT
 endif
 
 SYSTEMTEST_RESROOT=$(TEST_RESROOT)/../../system
+
+define JCK_CMD_TEMPLATE
+export JAVA_HOME=$(TEST_JDK_HOME) $(AND_IF_SUCCESS) perl $(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl \
```
Comment (llxia) [inline jck/jck.mk]: Since we already export `JAVA_HOME` in TKG, I do not think we need to export it here anymore.

Comment (Mesbah-Alam) [reply jck/jck.mk]: Makes sense. I'll remove the JAVA_HOME export and run a test after the current tests are done (which will take a little while).

Comment (Mesbah-Alam) [reply jck/jck.mk]: Removed export of JAVA_HOME. It works fine: Test_grinder/job/Grinder/6242/console

**Comment by Mesbah-Alam** [comment]
> Testing sanity.jck with jck11 on Linux: Test_grinder/job/Grinder/6235/console
Testing sanity.jck with jck14 on Windows: Test_grinder/job/Grinder/6239/console
Testing extended.jck with jck8 on s390 Linux: Test_grinder/job/Grinder/6238/console

The tests work fine. 

FYI @llxia

**Comment by llxia** [comment]
fyi @JasonFengJ9

**Pull request merged**